platform/mellanox/iproute2/scripts/get_sdk_patches.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python3
+
+'''
+Script to download SDK iproute2 patches.
+
+If we build SONiC with SDK debians being taken from github, need to downlaod patches from github, usage will be:
+./get_sdk_patches.py -i <iproute2 version> -s <SDK version>
+
+If we build SONiC with SDK compiled from sources, need to take patches from SDK sources, usage will be:
+./get_sdk_patches.py -i <iproute2 version> -u <SDK sources url> 
+'''
+import argparse
+import requests
+import sys
+import os
+import subprocess
+from zipfile import ZipFile
+
+SDK_GITHUB_URL_BASE = 'https://raw.githubusercontent.com/Mellanox/Spectrum-SDK-Drivers/'
+SDK_GITHUB_URL_BASE = 'https://raw.githubusercontent.com/liorghub/Spectrum-SDK-Drivers/'
+SDK_IPROUTE2_PATCHES = '/iproute2/patches/'
+SDK_CODE_PACKAGE = 'code_package'
+SDK_CODE_PACKAGE_ZIP = SDK_CODE_PACKAGE + '.zip'
+PATCH_DIR = 'patch'
+SERIES_FILE = 'series'
+
+
+def download_file(base_url, file_name):
+    url = base_url + '/' + file_name
+    response = requests.get(url)
+    if response.status_code != requests.codes.ok:
+        raise Exception("Failed to download {}".format(url))
+    open(file_name, "wb").write(response.content)
```

Comment:
wrap this with a context manager i.e. a `with` statement

Reply:
I have removed the script completely, now all operations are being performed from Makefile.